app/views/microposts/new.js.erb

```diff
@@ -1,2 +1,5 @@
-alert("Hii please proceed..");
-$("#blogs").append("<%=j render 'form', item: @micropost%>");
+$(document).ready(function(){
```

Comment:
@HrisikeshSinghaChowdhury why do we need this here?

Reply:
yes not required....